modules/navier_stokes/test/tests/bump/bump.i

```diff
@@ -111,32 +91,31 @@
     # realistic value.
     dynamic_viscosity = 0.0
     fluid_properties = ideal_gas
-  [../]
+  []
 []
 
 
 
 [Postprocessors]
-  [./entropy_error]
+  [entropy_error]
     type = NSEntropyError
     execute_on = 'initial timestep_end'
     block = 0
     rho_infty = 1.1768292682926829
     p_infty = 101325
     rho = rho
-    pressure = pressure
+    pressure = p
```

Comment:
This is where I'm confused. In `NS.h` you changed the pressure variable name to be "p" instead of "pressure", which necessitates this change here, which I'm hoping is the only reason you had to re-gold the corresponding gold file. Why?

Reply:
You've manually changed the CNS object files to have the explicit parameter name "pressure", so why be inconsistent and change the input file variable name (generated by the action) to "p"?

It's not ideal having INS use a "p" parameter name in lots of places, while we are using "pressure" in CNS; that's definitely inconsistent. But it seems like you're adding additional inconsistency here instead of removing it...?